packages/amplify-graphql-schema-generator/src/datasource-adapter/mysql-string-datasource-adapter.ts

```diff
@@ -279,12 +280,12 @@ export class MySQLStringDataSourceAdapter extends StringDataSourceAdapter {
     return value.match(regex).map((a) => a.slice(1, -1));
   }
 
-  private generateEnumName(tableName: string, fieldName: string) {
-    const enumNamePrefix = [tableName, fieldName].join('_');
+  private generateEnumName(tableName: string, fieldName: string): string {
+    const enumNamePrefix = toPascalCase([tableName, fieldName]);
     let enumName = enumNamePrefix;
     let counter = 0;
     while (this.enums.has(enumName)) {
-      enumName = [enumNamePrefix, counter.toString()].join('_');
+      enumName = toPascalCase([enumNamePrefix, counter.toString()]);
```

Comment:
Can you confirm this wouldn't be a breaking change if we change the naming of enums from `table_field` to `TableField`?

Reply:
As per the discussion @palpatim the change could affect customers only if they reimport the schema which is highly unlikely, but we still are waiting for conformation from Josef on the same